### ISSUE TITLE
How to run a second command when the first command fails? e.g., `tmux attach || tmux`

### ISSUE THREAD

**Issue by jcjolley** [nushell/nushell#5740]
In other shells, I would run `tmux attach || tmux` to resume a tmux session if one exists, or create one if it doesn't.

I crawled the docs and couldn't find an equivalent to this.
I read https://github.com/nushell/nushell/issues/740 and https://github.com/nushell/nushell/issues/1352, but neither of them addressed the OR case.

**Comment by fdncred** [comment]
you are correct. i don't believe we support it yet, but we want to.

**Comment by DominicDams** [comment]
I think a work around is using an if and complete
i.e. 
```
if ((tmux attach | complete | get exit_code) == 1) {
    tmux
}
```

But I agree this functionality would be nice

**Comment by dheater** [comment]
I started work on this so I'll go ahead and post a draft MR #6374

I've only got some initial parsing done. I've not figured out how to break the commands into conditionally executed blocks.

**Comment by fdncred** [comment]
Semi colon works now like `command;command2`

**Comment by dheater** [comment]
> Semi colon works now like command;command2

Yes, but that doesn't provide shortcut semantics so there is not a convenient way to do something like  `if file exists foo && ./foo`.
